hypha/settings/base.py

```diff
@@ -358,6 +358,10 @@
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = env.str("SOCIAL_AUTH_GOOGLE_OAUTH2_KEY", "")
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = env.str("SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET", "")
 
+# Use `SESSION_COOKIE_AGE_LONG` when a user authenticates with OAuth.
+# Will override `SOCIAL_AUTH_SESSION_EXPIRATION`
+SOCIAL_AUTH_USE_LONG_SESSION = env.bool("SOCIAL_AUTH_USE_LONG_SESSION", False)
```

Comment:
Could we make it so Hypha always use the long session time for oauth logins instead?

We could add  text to `SESSION_COOKIE_AGE_LONG` explaining that it is also used for oauth logins.

Reply:
that works, should be set now